app/lambda/create_health_care_professional/lambda_function.py

```diff
@@ -19,7 +19,7 @@ def putItem(self, HealthCareProfessionalName):
 
     def handle(self, event, context):
         try:
-            name = event[DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_NAME] if DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_NAME in event else None
+            name = event["queryStringParameters"][DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_NAME] if DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_NAME in event["queryStringParameters"] else None
             id = self.putItem(name)
```

Comment:
If there is no name, shouldnt this lambda  should return an error?

Reply:
Handled in datastore layer.

Reply:
Adding the error handling layer in the lambda also.